outline.md

```diff
@@ -23,46 +23,17 @@ While not claiming all of the answers, it summarizes the learnings of over a yea
 
 Participation in inner source requires real amounts of care and feeding to be effective.
 An inner source program is the set of tools, documentation, and training provided so that any team can participate in inner source effectively.
-To frame the talk, we'll explain the goals of an inner source program as well as the context within-which we've begun to pursue those goals.
-Even though the learning in this presentation is occurring in that context, we suspect that much of what's here applies generally.
 
 When building an inner source program you should have a vision of what you will provide to facilitate inner sourcing _as well as_ a strategy you will employ to roll out and make effective these services in your organization.
 We'll explain principles, resources, and specific examples that can aid and guide in doing so.
 
 The services of the inner source program should provide two things.
 First, they must teach, define, and document the type of culture that is conducive to inner source.
 Second, there must be software that reinforces that culture while simultaneously lowering the barrier to entry to participating in that culture.
```

Comment:
I think this looses some of its teeth without the concrete examples.  I'd propose mentioning at least one or two of the examples inline, specifically I'm thinking about the documentation portion.